src/test/java/seedu/address/testutil/CompanyBuilder.java

```diff
@@ -2,32 +2,30 @@
 
 import java.time.LocalDate;
 import java.time.format.DateTimeFormatter;
-import java.util.HashSet;
-import java.util.Set;
 
 import seedu.address.model.company.ApplicationStatus;
 import seedu.address.model.company.Company;
 import seedu.address.model.company.Deadline;
 import seedu.address.model.company.Email;
 import seedu.address.model.company.Name;
 import seedu.address.model.company.Phone;
+import seedu.address.model.company.Priority;
 import seedu.address.model.company.RecruiterName;
 import seedu.address.model.company.Role;
-import seedu.address.model.tag.Tag;
-import seedu.address.model.util.SampleDataUtil;
 
 /**
  * A utility class to help with building Company objects.
  */
 public class CompanyBuilder {
 
-    public static final String DEFAULT_NAME = "Twitter";
-    public static final String DEFAULT_PHONE = "85355255";
-    public static final String DEFAULT_EMAIL = "amy@gmail.com";
+    public static final String DEFAULT_NAME = "TWITTER";
```

Comment:
It matters if our company name is in Full Caps is it?

Reply:
No, it does not. I might have changed it on accident